engine/src/main/java/org/terasology/rendering/dag/tasks/BindFBOTask.java

```diff
@@ -38,4 +38,9 @@ public void execute() {
     public void setFboToBind(int fboToBind) {
         this.fboToBind = fboToBind;
     }
+
+    @Override
+    public String toString() {
+        return this.getClass().getSimpleName() + ": " + fboToBind;
```

Comment:
Hmmm. This will generate something such as:

`BindFBOTask:3`

It's a very spartan debug string. We won't be able to tie this back to a node nor (more importantly) to an fboName when we see the output of the taskListGenerator.

Either the BindFBO state change prints out both the fboName and the fboId or the task must store of the fboName, I think, purely for debugging purposes.